src/services/users.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable } from '@nestjs/common';
 import { EntityManager } from 'typeorm';
 import { Users } from '../entities/user.entity';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const bcrypt = require('bcrypt');
```

Comment:
We can import the relevant functions like so

```suggestion
import { genSalt, hash } from 'bcrypt';
```

Then we wouldn't need disable the linting rule here

Reply:
Same in the `src/services/auth/auth.service.ts` file

Reply:
Had to install a new [dev dependency](https://www.npmjs.com/package/@types/bcrypt) for that suggestion to work 🤷🏾‍♂️